webui/src/pages/user/add-namespace-member-dialog.tsx

```diff
@@ -124,11 +124,10 @@ export const AddMemberDialog: FunctionComponent<AddMemberDialoProps> = props =>
                     label='Open VSX User'
                     fullWidth
                     onChange={handleUserSearch}
-                    onKeyPress={(e: React.KeyboardEvent) => {
-                        if (e.charCode === 13) {
-                            if (foundUsers.length === 1) {
-                                addUser(foundUsers[0]);
-                            }
+                    onKeyPress={(e) => {
+                        if (e.key === "Enter" && foundUsers.length === 1) {
```

Comment:
Condensed the two `if` statements into one, I think readability isn't sacrificed although I can see a good argument for keeping these two separate :)